### ISSUE TITLE
oracledbinstall.js http proxy username/password not support

### ISSUE THREAD

**Issue by RaphaelMui** [oracle/node-oracledb#906]
I was trying to install the node_oracledb prebuild binaries.   My computer accesses the internet via http/https proxy which require username/password authentication.

My proxy settings are,
```
> export http_proxy=http://USERNAME:PASSWORD@PROXYSERVER.MYDOMAIN.COM:8080
> export https_proxy=http://USERNAME:PASSWORD@PROXYSERVER.MYDOMAIN.COM:8080
```

I found that the installation failed when running 'node package/oracledbinstall.js'  which tried to download oracledb-v2.2.0-node-v59-win32-x64.gz 
```
> node package/oracledbinstall.js
oracledb Beginning installation
oracledb Verifying installation
oracledb Continuing installation
oracledb ERR! NJS-054: Binary build/Release/oracledb.node was not installed.
oracledb ERR! Failed to install binary package oracledb-v2.2.0-node-v59-win32-x64.gz
oracledb ERR! Error: HTTP proxy request for github.com/oracle/node-oracledb/releases/download/v2.2.0/oracledb-v2.2.0-node-v59-win32-x64.gz failed with code 407
```

From the  source code of oracledbinstall.js,  it seems that http username and password are not supported.

1. What is your Node.js version?  Is it 64-bit or 32-bit?  Run `version.js` from https://github.com/oracle/node-oracledb/blob/master/examples/version.js
v9.11.1
2. What is your node-oracledb version?
2.2.0
3. What is your Oracle client (e.g. Instant Client) version?  Is it 64-bit or 32-bit?  How was it installed?  Where is it installed?
Instanctclient_12_2
4. What is your Oracle Database version?
11gR2
5. What is your OS and version?
Ubuntu 18.04

**Comment by Nebotov** [comment]
I have the same error

**Comment by cjbj** [comment]
@Nebotov code supporting this has been [merged to the master branch](https://github.com/oracle/node-oracledb/pull/919) but there has been no release yet - you will need to build it yourself in the meantime.

**Comment by RaphaelMui** [comment]
The problem solved in node-oracledb version 3.0
